grunt-tasks/jst.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function(grunt) {
+  grunt.config("jst", {
+    compile: {
+      options: {
+        namespace: "wcTmpl",
+        prettify: true,
+        processContent: function(src) {
+          // strip the opening and closing <script> tags...
+          // otherwise, the template functions will just inject script elements
+          // that won't render.
+          src = src.split(/\<script type=['"]text\/template['"]>/)[1];
```

Comment:
@miketaylr thanks for the explanation. regexp vs string.split. If the pattern is regular aka no variation I would have use a split. You could microbenchmark it too :) not sure if it's meaningful performance gain. 

oh it already exists. 
https://jsperf.com/regex-vs-split/2
and https://jsperf.com/slice-vs-substr-vs-substring-vs-split-vs-regexp/2